ini.c

```diff
@@ -156,6 +156,14 @@ int ini_parse_stream(ini_reader reader, void* stream, ini_handler handler,
 
         lineno++;
 
+#if INI_USE_STACK
+        if (strlen(line) == max_line - 1 && line[max_line - 2] != '\n') {
+            /* Exit even if INI_STOP_ON_FIRST_ERROR is not set */
```

Comment:
Yeah, I'm not sure about this. If `INI_STOP_ON_FIRST_ERROR` is false (the default), I think we should read and discard the rest of the current line, and then proceed. We could decide to do this as a stop-gap fix, but I'm not sure that's a good idea -- if we fix this I think we should do it properly in a single PR.

You can see how it's breaking the existing max-line tests as it is currently.